sdks/python/apache_beam/pvalue.py

```diff
@@ -248,19 +244,41 @@ def _view_options(self):
     """
     return {'window_mapping_fn': self._window_mapping_fn}
 
+  @property
+  def element_type(self):
+    return typehints.Any
+
+
+class AsSingleton(AsSideInput):
+  """Marker specifying that an entire PCollection is to be used as a side input.
```

Comment:
maybe a note on (The input PCollection should contain at most one element) in this comment block.